app/controllers/users_controller.rb

```diff
@@ -1,7 +1,57 @@
+# frozen_string_literal: true
+
 class UsersController < Clearance::UsersController
```

Comment:
```suggestion
class UsersController < ApplicationController
```

Reply:
We don't need to inherint from that controller. It contains logic that is not needed here

Reply:
Ready too 👍